src/seedu/addressbook/parser/Parser.java

```diff
@@ -135,6 +139,32 @@ private Command prepareAdd(String args) {
         }
     }
 
+    private Command prepareAddList(String args) {
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.